packages/qulacs/tests/qulacs/circuit/test_noise_model_qulacs.py

```diff
@@ -247,4 +249,6 @@ def test_convert_kraus_cptp() -> None:
         kdg = k.conj().T
         cptp += k @ rho @ kdg  # K_i rho K_i^dagger
 
-    assert np.allclose(cptp, matrix.get_matrix())
+    # the `.get_matrix()` method returns NDArray[complex128, _Shape]
+    # (_Shape should be the first argument)
```

Comment:
I think this is clearer:
```suggestion
    # We need to disable type check due to an error in qulacs type annotation
    # https://github.com/qulacs/qulacs/issues/537
```